src/openrct2/CmdlineSprite.cpp

```diff
@@ -185,7 +185,7 @@ bool SpriteFile::Save(const utf8* path)
 static bool SpriteImageExport(const rct_g1_element& spriteElement, const char* outPath)
 {
     const auto pixelBufferSize = spriteElement.width * spriteElement.height;
-    std::unique_ptr<uint8_t[]> pixelBuffer(new uint8_t[pixelBufferSize]);
+    std::unique_ptr<uint8_t[]> pixelBuffer = std::make_unique<uint8_t[]>(pixelBufferSize);
```

Comment:
Maybe use `auto` on the left hand side now? (No idea why GH is not allowing me to suggest it)

(Applies to all of https://github.com/OpenRCT2/OpenRCT2/pull/15249/commits/dee05d7edb25630c5fb69cfd4ad33d5f4ae3ed35)